packages/zoe/src/contractSupport/ratio.js

```diff
@@ -22,6 +23,15 @@ const { multiply, floorDivide, add, subtract } = natSafeMath;
 // brand-checking helps us ensure that normal Ratios aren't applied to amounts
 // of the wrong brand, and that exchange rates are only used in the appropriate
 // direction.
+//
+// Since the ratios are represented by a numerator and a denominator, every
+// multiplication or division operation that produces an amount ends with a
+// division of the underlying bigIntegers, and bigInteger division requires that
```

Comment:
Is "bigInteger" a thing people will recognize? Or is "bigint" clearer? It made me do a double-take

Reply:
Good point. I see it almost universally as `BigInt` and very occasionally as `bigint`. Let's change to that.